packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -0,0 +1,32 @@
+# WebDriverIO Testing
+
+The terra-functional-testing library is a utility for developing tests using the [WebdriverIO Framework](https://webdriver.io/).
+
+## Accessibility Testing
+
+The testing library integrates [axe-core](https://github.com/dequelabs/axe-core) utilities for automating accessibility testing.
```

Comment:
```suggestion
The testing library integrates [axe-core](https://github.com/dequelabs/axe-core) utilities for automating accessibility testing of the current dom composition.
```

Reply:
Can we add docs on what axe rules we opt-in and test?

Reply:
Updated the docs to add more information on rules and how axe runs on the document:

https://github.com/cerner/terra-toolkit/pull/452/commits/1aad2e2c77dc3a59fed32b3ce8c763a95b57d952